### PR TITLE
[shopsys] resolve several Symfony deprecations

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| We want to upgrade Symfony to 5 LTS version so we need to get rid of 4.4 deprecations
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| Yes 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/listGrid.html.twig [hunk 1]
```diff
@@ -14,4 +14,4 @@
 {% block grid_pager_totalcount %}
     {% set entityName = 'category'|trans %}
     {{ parent() }}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```
Comment (grossmannmartin) [inline packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/listGrid.html.twig:17]: do you know where is this template used? I did not find the usage...

File: phpstan.neon [hunk 2]
```diff
@@ -9,9 +9,6 @@ parameters:
 
     ignoreErrors:
         # shopsys/framework - don't forget to add these rules to phpstan.neon in framework
-        -
-            message: '#^Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition::children\(\)\.$#'
-            path: %currentWorkingDirectory%/packages/framework/src/Component/Domain/Config/DomainsUrlsConfigDefinition.php
```
[the remark this answers is not in the document]
Comment (vitek-rostislav) [reply phpstan.neon:14]: ah, yeah, it was there with a different pattern so I missed it the first time, fixed in 1e312257c :+1:

File: packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/listGrid.html.twig (hunk 1 above)
Comment (vitek-rostislav) [reply packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/listGrid.html.twig]: that is a very good question :smile: even the content is somewhat weird, I removed it in 88aba0c18

**Review by grossmannmartin — APPROVED** [review]
nice work

**Pull request merged**